qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VisitTypeSteps.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertTrue;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.helper.RestClient;
+import org.openmrs.contrib.qaframework.page.AdministrationPage;
+import org.openmrs.contrib.qaframework.page.VisitTypeListPage;
+import org.openmrs.contrib.qaframework.page.VisitTypePage;
+
+public class VisitTypeSteps extends Steps {
+
+    private AdministrationPage administrationPage;
+    private VisitTypePage visitTypePage;
+    private VisitTypeListPage visitTypeListPage;
+    private String visitTypeUuid;
+    private final String VISIT_TYPE_NAME = RandomStringUtils.randomAlphabetic(5);
+    private static final String VISIT_TYPE_DESCRIPTION = "Description for Visit Type test workflow";
+
+    @Before(RunTest.HOOK.SELENIUM_VISIT_TYPE)
+    public void setup() {
+        initiateWithLogin();
+    }
+
+    @After(RunTest.HOOK.SELENIUM_VISIT_TYPE)
+    public void teardown() {
+        RestClient.delete("visittype/" + visitTypeUuid, true);
+        quit();
+    }
+
+    @Given("a user clicks on the system administration link from the home page")
+    public void launchSystemAdministrationPage() {
+        administrationPage = homePage.goToAdministration();
+    }
+
+    @Then("the system loads the administration page")
```

Comment:
Quick solution would be simply changing the naming of these classes. or change your scenario naming to something like
```suggestion
    @Then("  administration page gets loaded")
    
    And let us see what happens after pushing.
```